swagger_server/utils/db_utils.py

```diff
@@ -1,64 +1,51 @@
 import logging
 import os
 
-import _pickle as pickle
-import dataset
+import pymongo
 
+DB_NAME = os.environ.get("DB_NAME")
 DB_CONFIG_TABLE_NAME = os.environ.get("DB_CONFIG_TABLE_NAME")
```

Comment:
Like I mentioned in https://github.com/atlanticwave-sdx/sdx-controller/pull/126 review comments, we should get the terminology right: Mongo has documents and collections, and tables is relational database terminology.

Reply:
The MongoDB terms are too specific to the mongo world. I'm trying to stick with a more general API terms, so in the future if anything changes, no need to update the terms. The DBUtils is abstracting the db access from the rest, so user don't need to get used to different db terms.